CardsCashCasino/Manager/TexasHoldEmManager.cs

```diff
@@ -583,80 +666,104 @@ public void DealRiver()
         /// Winner is declared for the round
         /// </summary>
         public void RoundConclusion()
-        {   
+        {
             PokerUtil.Ranking bestRanking = PokerUtil.Ranking.HIGH_CARD;
             // List of hands that (so far) are tied for the best rank.
             // Pair of player idx and their optimal 5-card hand. 
             List<Tuple<int, List<Card>>> bestHands = new();
-            // For each player
-            for (int i = 0; i < _playerHands.Count; i++)
+            List<int> eligiblePlayers = new();
+            for (int potNumber = 0; potNumber < _potManager.Pots.Count; potNumber++)
             {
-                // Reveal the player's cards
-                _playerHands[i].Cards[0].GetTexture();
-                _playerHands[i].Cards[1].GetTexture();
-
-                // Get ranking and optimal hand 
-                Tuple<List<Card>, PokerUtil.Ranking> pair = PokerUtil.GetScore(_communityCards, _playerHands[i].Cards.ToList());
-                // If this ties the best ranking
-                if (pair.Item2 == bestRanking)
+                eligiblePlayers = _potManager.PlayersEligible(potNumber);
```

Comment:
confused by this line. we define the list of eligible players before the for loop, then immediately reassign it here. is there a reason we need to ever initialize it before the for loop?